doc/deployment_guide.md

```diff
@@ -49,5422 +49,6234 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
+
+The Dell EMC Ready Architecture for Red Hat OpenStack Platform uses the S5248-ON as the Top of Rack switches and the S3048-ON switch (S4048-ON optional) as the management switch.
+
+**Infrastructure layouts**
+
+The network consists of the following major network infrastructure layouts:
+
+* Core Network Infrastructure - The connectivity of aggregation switches to the core for external connectivity.
+* Data Network Infrastructure - The server NICs, top-of-rack (ToR) switches, and the aggregation switches.
+* Management Network Infrastructure - The BMC management network, consisting of iDRAC ports and the out-of-band management ports of the switches, is aggregated into a 1-rack unit (RU) S3048-ON switch in one of the three racks in the cluster. This 1-RU switch in turn can connect to one of the aggregation or core switches to create a separate network with a separate VLAN.
+
+
+> Note: Please contact your Dell EMC sales representative for a detailed parts list or contact Dell EMC Professional Services team for OpenStack using the email [rhosp.ra.ps@emc.com](malito:rhosp.ra.ps@emc.com)
+
+### Version 13.2 features
+
+**The following features are added in the JetPack Automation and the Ready Architecture for RedHat OpenStack:**
+
+* Support for the lastest release of Red Hat OpenStack Platform 13.2 including the latest updates.
+* Support for latest release of RHEL 7.7 including the latest updates.
+* Support for Red Hat Ceph Storage version 3.2.2.
+* Added support for Jumbo Frames.
+* Added support for UEFI on all Overcloud nodes.
+* Added support for Dell EMC Networking S5248-ON switch.
+* Added support for 25GbE networking with Intel XXV710 network interface cards.
+* Added support for 100GbE networking with Mellanox ConnectX-5 network interface cards.
+* Added support for Red Hat Ceph Storage to use NVMe storage for OSD/Journal.
+* Enhancement of Huge Pages with JetPack CSP profile.
+* Enhancement of NUMA/CPU Pinning with JetPack CSP profile.
+* Added support of OVS-DPDK for 100G networking with Mellanox network interface cards (NICs).
+* Added support of SR-IOV offload for 100G networking with Connect X-5 Mellanox network interface cards (NICs) (Tech Preview in RedHat)
+* Added support of SR-IOV for 25GbE networking with Intel XXV710 network interface cards.
+* Added support for Cascade Lake 6230 processors
+* Added support for Distributed Virtual Router, (DVR).
+* Added support for auto-generation of .ini and .properties files for use in automated deployments.
+* Added support for VLAN aware VM.
+* Added support of Unity with Manila and Cinder
+* Added support of Octavia with Jetpack CSP profile
+* Added support of Barbican with Jetpack CSP profile
+* Added support of Satellite with Jetpack CSP profile
+* Added support for Bluestore
+
```

Comment:
Feature list of what s new above is wrong, see latest RA doc for the feature list

Reply:
Can you please share the link for latest RA doc? we are not able to find it here: https://www.dell.com/support/article/hr/en/hrbsdt1/sln310368/dell-emc-ready-architecture-for-red-hat-openstack-platform?lang=en